### ISSUE TITLE
Alt text should allow more characters than 420

### ISSUE THREAD

**Issue by dilinger** [nolanlawson/pinafore#2336]
I posted a screenshot that included a post thread. I wanted to paste the actual thread text into the alt text, but there wasn't enough space. I found that kind of odd, until I realized that pinafore only allows 420 characters of alt text; that's actually less than the post limit of 500 characters!

Meanwhile, my local mastodon instance running 4.0.2 allows 1500 characters of alt text. Please consider updating pinafore to allow 1500 characters for alt text as well.

It's especially important for people with disabilities that images of text include the actual text. For example, if you're posting a screenshot of a community notice showing covid testing site locations, there should be enough alt text space for the person posting to actually cut & paste the information. People with screenreaders should be able to discover the locations, and not just see a summary of the image saying "An image of covid testing site locations". For that, more characters than 420 is necessary.

**Comment by ChrisDuffley** [comment]
In other words, I also suggest that, like how character limits for toots are honored, the info for the alt text limit that's server-side should also be used, rather than being limited client-side.

**Comment by nolanlawson** [comment]
Thanks for the suggestion. I set it to 420 because this was (at least at the time) the limit that Mastodon imposes. If your instance has customized this behavior, then is there some way it is communicated?

For instance, the maximum number of characters in a post (default 500) is communicated via `configuration.statuses.max_characters` in the API (e.g. https://mastodon.social/api/v1/instance). However I don't see any equivalent one for alt text, at least on `mastodon.social`.
